receiver/hostmetricsreceiver/internal/scraper/processscraper/internal/metadata/generated_metrics.go

```diff
@@ -48,7 +49,53 @@ type MetricsSettings struct {
 	ProcessThreads             MetricSettings `mapstructure:"process.threads"`
 }
 
+func DeafultDarwinMetricsSettings() MetricsSettings {
```

Comment:
This file is autogenerated, we should not modify it. cc @dmitryax, what would be the proper way to handle this?

Reply:
tested again to remove changes done in that file and should be good now